Core/Entities/Sys_OtherInformation.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Core.Entities
 {
-    [Table("tbl_Other_Information")]
+    [Table("tbl_OtherInformation")]
     public class Sys_OtherInformation : BaseEntity, IAuditableEntity
```

Comment:
Have you fire query for this in mssql too because i saw migration not working on our project